docs/source/usage.rst

```diff
@@ -35,12 +35,14 @@ A simple connection with BigchainDB can be established like this.
 
 	const conn = new driver.Connection(API_PATH)
 
-It is also possible to connect to a BigchainDB node of the IPDB test net.
-To do so, you need to pass the **app_id and app_key**.
+Previously you could connect to IPDB testnet, but it has been shutdown.
```

Comment:
We don't need to explain about IPDB history. I would say that we are connection to BigchainDB node.